sdk/communication/Azure.Communication.CallAutomation/src/Models/Events/RecognizeDtmfCompleted.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Azure.Core;
+using System.Text.Json.Serialization;
+using System;
+
+namespace Azure.Communication.CallAutomation.Models.Events
+{
+    /// <summary>
+    /// The Recognize Dtmf Completed event.
+    /// </summary>
+    public class RecognizeDtmfCompleted : RecognizeCompleted
+    {
+        /// <summary> Get the recognize Tone result. </summary>
+        public CollectTonesResult CollectTonesResult { get; }
+        /// <summary>
+        /// The recognition type.
+        /// </summary>
+        [CodeGenMember("RecognitionType")]
+        [JsonConverter(typeof(EquatableEnumJsonConverter<CallMediaRecognitionType>))]
+        public CallMediaRecognitionType RecognitionType { get; set; }
```

Comment:
Internal set

Reply:
do you mean set the recognitionType as private field?